### ISSUE TITLE
The plugin is broken in Sublime 2.0.2

### ISSUE THREAD

**Issue by jfjauvin** [Zeeker/sublime-GitConflictResolver#10]
I decided to try your plugin.

Downloaded the latest with Package Control (1.4.7) and got the following error in the console:

```
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 62, in reload_plugin
  File ".\GitConflictResolver.py", line 5, in <module>
    from .modules import conflict_re
ValueError: Attempted relative import in non-package
```
I looked at the changelog and saw this might be related to your refactoring, so I downloaded release (1.4.6) from GitHub and then got some "object has no attribute 'DRAW_NO_FILL'" which I presume is a new setting in Sublime 3.

Anyhow, I decided to quit there and do my merge manually.

**If the plugin is intended for Sublime 3 only**, please specify it somewhere.

**Comment by Asuza** [comment]
This looks like the exact same problem I'm having. It was working fine and then just quit working. It must have been when I upgraded to Sublime Text 2.0.2. I'm seeing the same error that @jfjauvin mentioned.

**Comment by Zeeker** [comment]
Fixed in release 1.4.8 (fe331c5cb774f2da37a3ae8d772b181156d61a81)

Sorry that it took so long!
